pkg/networks/usernet/client.go

```diff
@@ -74,11 +75,15 @@ func (c *Client) ResolveAndForwardSSH(ipAddr string, sshPort int) error {
 }
 
 func (c *Client) ResolveIPAddress(ctx context.Context, vmMacAddr string) (string, error) {
-	timeout := time.After(2 * time.Minute)
+	timeout := 2 * time.Minute
+	if os.Getenv("GITHUB_ACTIONS") != "" {
```

Comment:
@afbjorklund Did you mean this?

Reply:
Yeah, if it should be 3 (or 5) minutes everywhere ? And what is taking so long

Reply:
system resource contention?